raiden/ui/cli.py

```diff
@@ -237,7 +237,7 @@ class AddressType(click.ParamType):
     def convert(self, value, param, ctx):
         try:
             return address_decoder(value)
-        except BadRequestError:
+        except (BadRequestError, TypeError):
```

Comment:
BadRequestError doesnt seem to be used anywhere, I think we should remove it.

Reply:
Also, consider changing this to a LBYL style, e.g.:

```python
if value:
  return address_decoder(value)
self.fail('Please specify a valid hex-encoded address.')
```

Reply:
But we still need to catch the TypeError, right? Then it doesn't look better than before.

```py
if value:
  try:
    return address_decoder(value)
  except TypeError:
    self.fail('Please specify a valid hex-encoded address.')
self.fail('Please specify a valid hex-encoded address.')
```

Reply:
I'm assuming the type error is because `value` is `None`, so the exception wouldn't exist in that case. Am I wrong? What is the value that resulted in the exception?

Reply:
There are other reasons as well, such as having non-hex strings or to short addresses that give TypeErrors. I tested with `--address=test` and `--address=not-an-address` for these cases.

Reply:
ah, I see, thanks